### PR TITLE
serializer submaterials of multimaterial

### PR DESCRIPTION
**Pull request by kcoley**
- Serialize sub materials of multimaterial

### CODE REVIEW

File: src/Tools/babylon.sceneSerializer.ts [hunk 1]
```diff
@@ -67,10 +67,17 @@
             if (mesh.material) {
                 if (mesh.material instanceof MultiMaterial) {
                     serializationObject.multiMaterials = serializationObject.multiMaterials || [];
+                    serializationObject.materials = serializationObject.materials || [];
                     if (!serializationObject.multiMaterials.some((mat: Material) => (mat.id === (<Material>mesh.material).id))) {
                         serializationObject.multiMaterials.push(mesh.material.serialize());
+                        for (let submaterial of mesh.material.subMaterials) {
+                            if (submaterial instanceof Material) {
```
Comment (deltakosh) [inline src/Tools/babylon.sceneSerializer.ts]: Why this test?

Comment (kcoley) [reply src/Tools/babylon.sceneSerializer.ts]: This was to check for nullable, but I can change it to see if it is not null

Comment (kcoley) [reply src/Tools/babylon.sceneSerializer.ts]: Just made the change

**Comment by deltakosh** [comment]
Great ! thanks!

**Pull request merged**